views/runsui.pug

```diff
@@ -18,11 +18,10 @@ block content
         div.form-group.row
           label(for="taginput") Add tags to selected: &nbsp;
           select.form-control#taginput(style="width:20%;min-width:150px") Choose a tag
+            option(value="prioritize") prioritize
             option(value="messy") messy
-            option(value="bad") bad
+            option(value="abandon!!!11one1!!!1") abandon
```

Comment:
can we not do this?

Reply:
I blame @WenzDaniel

Reply:
:D

Reply:
I would have to add this to a bootstrax query, my eyes hurt to see this, perhaps we should have a dedicated NV tag where we can have Daniel have his moment of fun